packages/react-scripts/template-typescript/.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+Fixes:
+- [issue or a JIRA task]
+
+Pull request checklist:
+- [ ] Meaningful and properly prefixed title describing either which issue has been fixed (prefix `Fix:`), which feature has been added (`Feature: `) or what has been improved in the build process (`Tools:`)
```

Comment:
I would rather stick with natural language: `Add feature X to component Y`. `Feature:` can be added via tag

Reply:
We plan to experiment with automatic changelog generation basing on pull request titles, hence this approach.